GoogleUtilities.podspec

```diff
@@ -37,6 +37,15 @@ other Google CocoaPods. They're not intended for direct public usage.
     ls.dependency 'GoogleUtilities/Environment'
   end
 
+  s.subspec 'FileDataStorage' do |ls|
```

Comment:
Let's add the class to `Environment` subspec to reduce amount of subspecs.
cc @paulb777 @ryanwilson